src/game/Accounts/AccountMgr.cpp

```diff
@@ -48,7 +52,9 @@ AccountOpResult AccountMgr::CreateAccount(std::string username, std::string pass
         return AOR_NAME_ALREADY_EXIST;                       // username does already exist
     }
 
-    if (!LoginDatabase.PExecute("INSERT INTO account(username,sha_pass_hash,joindate) VALUES('%s','%s',NOW())", username.c_str(), CalculateShaPassHash(username, password).c_str()))
+    sha_pass_hash = CalculateShaPassHash(username, password);
```

Comment:
See earlier remark: ```std::string sha_pass_has = CalculateShaPassHash(username, password);```.

Reply:
since I rebased the code `sha_pass_hash` variable is obsolete and dropped